src/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -262,48 +263,100 @@ internal IPAddress(int newAddress)
         /// </devdoc>
         public static bool TryParse(string ipString, out IPAddress address)
         {
-            address = IPAddressParser.Parse(ipString, true);
+            if (ipString == null)
+            {
+                address = null;
+                return false;
+            }
+
+            const bool tryParse = true;
+            address = IPAddressParser.Parse(ipString, tryParse);
```

Comment:
I should have been clearer about naming the argument.  I meant doing it like this:
```C#
address = IPAddressParser.Parse(ipString, tryParse: true)
```